scripts/travis.sh

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env bash
```

Comment:
apollo.sh always source apollo_base.sh which needs user to input "y or Y" to accept the license.